elements/pfe-tabs/src/pfe-tab.scss

```
@@ -174,7 +175,7 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, $backwards-compatibility);
 
 // nth of type is needed to override first and last of type selectors above
 :host([pfe-variant="earth"][aria-selected="true"]) {
-  background-color: pfe-var(surface--lightest);
+  background-color: pfe-local(BackgroundColor, $fallback: #{pfe-var(surface--lightest)});
```

```suggestion
  background-color: pfe-local(BackgroundColor, $fallback: pfe-var(surface--lightest));
```

Does not need interpolation though 🥳